tsconfig.json

```diff
@@ -1,7 +1,8 @@
 {
   "compilerOptions": {
     "esModuleInterop": true,
-    "noEmit": true
+    "noEmit": true,
+    "strictNullChecks": true
```

Comment:
Huh... I thought this was covered by `strict`.

Reply:
`@typescript-eslint` was complaining without this.

Reply:
This is covered by `strict`, but we don't have `strict` enabled here. It's just in the packages tsconfig.

Reply:
Ah. Makes sense.